compiler/luci/import/src/Nodes/CircleLQFullyConnected.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Import/Nodes/CircleLQFullyConnected.h"
+#include <luci/IR/Nodes/CircleLQFullyConnected.h>
+
+#include <loco.h>
+
+namespace luci
+{
+
+bool CircleLQFullyConnectedGraphBuilder::validate(const ValidateArgs &args) const
+{
+  return GraphBuilder::validate(args, 5);
+}
+
+CircleNode *CircleLQFullyConnectedGraphBuilder::build_node(const circle::OperatorT &op,
+                                                           const std::vector<CircleNode *> &inputs,
+                                                           loco::Graph *graph) const
+{
+  auto *node = graph->nodes()->create<CircleLQFullyConnected>();
+
+  node->input(inputs.at(0));
+  node->input_scales(inputs.at(1));
+  node->weights_scales(inputs.at(2));
+  node->weights_binary(inputs.at(3));
+  node->bias(inputs.at(4));
```

Comment:
This order must be similar to order from node definition in lang:

https://github.com/Bronnikoff/ONE/blob/46a4c3de8aab49d0e3956891cca10cb2780d9b1c/compiler/luci/lang/include/luci/IR/Nodes/CircleLQFullyConnected.h#L38-L58

Reply:
Can you clarify?

Reply:
I mean indeces of input nodes for LQFullyConnected must be same in both files.